src/clj/rems/css/styles.clj

```diff
@@ -272,21 +292,60 @@
                      :color "#ccc"}
                     [:&:hover {:color (util/get-theme-attribute :color4)
                                :background-color "#eee"}]]]
-   [:.alert-info
-    (s/descendant :.state-info :.phases :.phase.completed)
-    {:color (util/get-theme-attribute :info-color)
-     :background-color (util/get-theme-attribute :info-bgcolor)}]
-   [:.alert-success
-    (s/descendant :.state-approved :.phases :.phase.completed)
-    {:color (util/get-theme-attribute :success-color)
-     :background-color (util/get-theme-attribute :success-bgcolor)}]
-   [:.alert-warning {:color (util/get-theme-attribute :warning-color)
-                     :background-color (util/get-theme-attribute :warning-bgcolor)}]
-   [:.alert-danger
-    :.state-rejected
-    (s/descendant :.state-rejected :.phases :.phase.completed)
-    {:color (util/get-theme-attribute :danger-color)
-     :background-color (util/get-theme-attribute :danger-bgcolor)}]
+
+   (when-let [style (remove-nil-vals {:color (util/get-theme-attribute :text-primary)})] [:.text-primary style])
```

Comment:
This unfortunately complicates reading the styles a lot. I think mostly having a default value for the themeable values would've been better. Btw. It can be "default" or "inherit" too!

Reply:
```
[:.text-primary {:color (util/get-theme-attribute :text-primary "default")}]
```

Reply:
Also you could extract the whole theme getter call to the beginning of the file and refer to it as `text-primary-color`.

Reply:
I can do this refactoring later if you want this merged soon.

Reply:
The `when-let` stuff can be avoided by just returning `{}` from `remove-nil-vals` if all properties are removed.

Reply:
Or actually it doesn't matter whether it returns `{}` or `nil`. The CSS minifier removes the declaration automatically when it contains no properties.